src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,33 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../services/user';
+import { User } from '../../types/User';
+
+type Props = {
+  todo: Todo;
+  setTodo: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, setTodo }) => {
+  const [user, setUser] = useState<User>();
```

Comment:
The `user` state is initialized without a default value. It's better to initialize it with `null` to avoid potential undefined behavior.